README.md

```diff
@@ -1,2 +1,8 @@
 # GitHubAiAPI
 This is Example to test AI plugin for automatic code review
+
```

Comment:
The description 'This is Example to test AI plugin for automatic code review' could be improved. Consider providing more details about the purpose and functionality of the AI plugin.